x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/SearchableSnapshotAction.java

```diff
@@ -437,6 +466,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         if (totalShardsPerNode != null) {
             builder.field(TOTAL_SHARDS_PER_NODE.getPreferredName(), totalShardsPerNode);
         }
+        if (numberOfReplicas != 0) {
+            builder.field(NUMBER_OF_REPLICAS.getPreferredName(), numberOfReplicas);
+        }
```

Comment:
This is part of why it should be `@Nullable` and optional, here we "clobber" it so that the in and out would be different:

```json
"searchable_snapshot": {
  "total_shards_per_node": 1,
  "number_of_replicas": 0
}
```

in would result in:

```json
"searchable_snapshot": {
  "total_shards_per_node": 1
}
```

out, and makes modifying default values for phases more difficult in the future.